test/integration.test.js

```diff
@@ -12,6 +14,7 @@ const {
 
 const Server = () => {
   const server = Express();
+  server.use(CookieParser('secret'));
```

Comment:
Please use a random string here so we know that the exact string doesn't really matter for the purposes of testing.